src/components/ToggleMenu.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import { Link } from 'react-router-dom';
+import MenuIcon from '@material-ui/icons/Menu';
+import HomeIcon from '@material-ui/icons/Home';
+import BarChartIcon from '@material-ui/icons/BarChart';
```

Comment:
I'm  not certain (because each library adds/doesn't add this capability depending on the team, but, try and group these imports into something like this 

```tsx
import { MenuIcon, HomeIcon, BarChartIcon } from '@material-ui/icons';
```

And see if it still works.